packages/node/src/methods/state-channel/create/controller.ts

```diff
@@ -86,23 +105,28 @@ async function deployMinimumViableMultisigAndGetAddress(
     signer
   );
 
-  return new Promise(async (resolve, reject) => {
-    try {
-      proxyFactory.once("ProxyCreation", async proxy => {
-        resolve(proxy);
-      });
-
-      // TODO: implement retry around this with exponential backoff on the
-      // gas limit to increase probability of proxy getting created
-      await proxyFactory.functions.createProxy(
-        multisigMasterCopyAddress,
-        new Interface(MinimumViableMultisig.abi).functions.setup.encode([
-          multisigOwnerAddresses
-        ]),
-        { gasLimit: CREATE_PROXY_AND_SETUP_GAS }
+  try {
+    // TODO: implement retry around this with exponential backoff on the
+    // gas limit to increase probability of proxy getting created
+    const transaction: TransactionResponse = await proxyFactory.functions.createProxy(
+      multisigMasterCopyAddress,
+      new Interface(MinimumViableMultisig.abi).functions.setup.encode([
+        multisigOwnerAddresses
+      ]),
+      { gasLimit: CREATE_PROXY_AND_SETUP_GAS }
```

Comment:
I'm not sure where this variable is defined but we should probably create a "gas estimates" file in the codebase.

Reply:
Will address as a follow up since the scope of that is very clear.